src/openrct2/ride/Track.h

```diff
@@ -526,10 +526,10 @@ namespace TrackElemType
 
 enum
 {
-    TRACK_SEQUENCE_FLAG_DIRECTION_0 = (1 << 0),
-    TRACK_SEQUENCE_FLAG_DIRECTION_1 = (1 << 1),
-    TRACK_SEQUENCE_FLAG_DIRECTION_2 = (1 << 2),
-    TRACK_SEQUENCE_FLAG_DIRECTION_3 = (1 << 3),
+    TRACK_SEQUENCE_FLAG_DIRECTION_0 = (1 << 0),      // Huts/path connect to front
```

Comment:
We don’t use “entrance hut” - that’s an NE’ism. We use “entrance” or “entrance building”.